english-spanish/Part 2/english-spanish-Part 2-Family 2.csv

```diff
@@ -1,11 +1,12 @@
-italiano,Italian
-su,his
+italiano,Italian (m)
+su,his/her
 hijos,children (sons)
-pequeño,small
-simpática,nice
-alto,tall
-bajo,short
+pequeño,small (m)
+simpático,nice (m)
```

Comment:
I appreciate your work on this! would you mind changing this back to feminine so that there's still a good mix of seeing adjectives as either gender?